From ffff6a364972b237f05631df24b18ce0b2bbe852 Mon Sep 17 00:00:00 2001
From: Paul Keith <javelinanddart@gmail.com>
Date: Wed, 27 Jun 2018 01:00:02 +0200
Subject: [PATCH 08/12] Settings: Cleanup SimSettings additions

* This shouldn't be an overlay, and is resulting in dead code

Change-Id: I86bef790463ff5d40541f849f2d5e9fc1ae238c2
---
 res/values/cm_strings.xml                     |  2 --
 src/com/android/settings/sim/SimSettings.java | 17 +----------------
 2 files changed, 1 insertion(+), 18 deletions(-)

diff --git a/res/values/cm_strings.xml b/res/values/cm_strings.xml
index 267f417ea5..7d14568e18 100644
--- a/res/values/cm_strings.xml
+++ b/res/values/cm_strings.xml
@@ -148,8 +148,6 @@
     <string name="sim_disabled">disabled</string>
     <string name="sim_missing">missing or faulty</string>
     <string name="sim_enabler_need_disable_sim">SIM card will be deactivated. Do you want to continue?</string>
-    <string name="sim_enabler_will_disable_sim_title">Attention</string>
-    <string name="sim_enabler_need_switch_data_service">This SIM will be disabled and SIM <xliff:g id="slotid">%1$s</xliff:g> will be used for data services. Are you sure you want to continue?</string>
     <string name="sim_enabler_airplane_on">Unable to perform the operation while airplane mode is on.</string>
     <string name="sim_enabler_in_call">Unable to perform the operation while in call.</string>
     <string name="sim_enabler_both_inactive">Can\'t disable all SIM cards</string>
diff --git a/src/com/android/settings/sim/SimSettings.java b/src/com/android/settings/sim/SimSettings.java
index 8bd120ac5e..ba70d9508c 100644
--- a/src/com/android/settings/sim/SimSettings.java
+++ b/src/com/android/settings/sim/SimSettings.java
@@ -656,22 +656,7 @@ public class SimSettings extends RestrictedSettingsFragment implements Indexable
 
             switch(dialogId) {
                 case CONFIRM_ALERT_DLG_ID:
-                    String message;
-                    if (mContext.getResources().getBoolean(
-                            R.bool.confirm_to_switch_data_service)) {
-                        if (SubscriptionManager.getDefaultDataSubscriptionId() ==
-                                mSubInfoRecord.getSubscriptionId()) {
-                            message = mContext.getString(
-                                    R.string.sim_enabler_need_switch_data_service,
-                                    getProvisionedSlotId());
-                        } else {
-                            message = mContext.getString(R.string.sim_enabler_need_disable_sim);
-                        }
-                        builder.setTitle(R.string.sim_enabler_will_disable_sim_title);
-                    } else {
-                        message = mContext.getString(R.string.sim_enabler_need_disable_sim);
-                    }
-                    builder.setMessage(message);
+                    builder.setMessage(R.string.sim_enabler_need_disable_sim);
                     builder.setPositiveButton(android.R.string.ok, mDialogClickListener);
                     builder.setNegativeButton(android.R.string.no, mDialogClickListener);
                     builder.setOnCancelListener(mDialogCanceListener);
-- 
2.17.1

